spec/controllers/users_controller_spec.rb

```diff
@@ -216,6 +216,12 @@
       get 'privacy_settings'
       expect(response.status).to eq(200)
     end
+
+    it "updates chat privacy settings" do
+      put(:update_privacy_settings, user: {disable_chat_login: true})
+      expect(request.flash[:notice]).to eql(I18n.t("users.update.settings_updated"))
+      expect(request.flash[:error]).to be_blank
+    end
```

Comment:
Please check that this actually changes the privacy settings. (both true -> false and false -> true)